cmd/tracee-rules/input.go

```diff
@@ -52,7 +52,7 @@ func setupTraceeGobInputSource(opts *traceeInputOptions) (chan protocol.Event, e
 	res := make(chan protocol.Event)
 	go func() {
 		for {
-			var event trace.Event
+			var event protocol.Event
```

Comment:
I might be missing something. If event is not sent in "protocol format", will we fail in decoding? is this intentional?

Reply:
Yes. Notice that before we would fail on any event that wasn't a `trace.Event` so this is an improvement.